baseband/mark4/base.py

```diff
@@ -211,6 +221,21 @@ def find_header(self, template_header=None, ntrack=None, decade=None,
         self.fh_raw.seek(offset)
         return header
 
+    @staticmethod
+    def ref_time_to_decade(ref_time, header_unit_year):
```

Comment:
I think this should be a private function.

Also, might it make more sense to pass in the relevant header and the ref_time? In that case, it would be easier to use in both `read_frame` and `find_header`.

Finally, perhaps silly, but now that I look at this again, I wonder if it doesn't below to `Mark4Header` after all. Not as a separate argument for `__init__`, which I think was bad, but perhaps by making `decade` a property and defining a setter that, e.g., checks whether was is passed in is a `Time` instance (or anything that has a `.decimalyear` attribute).